### PR TITLE
Add a CLI client

### PR DESCRIPTION
**Pull request by lziest**

### CODE REVIEW

File: cmd/ro/main.go [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+	flag.StringVar(&userEnv, "userenv", "RO_USER", "env variable for user name")
+	flag.StringVar(&pswdEnv, "pswdenv", "RO_PASS", "env variable for user password")
+}
+
+func getUserCredentials() {
+	user = os.Getenv(userEnv)
+	pswd = os.Getenv(pswdEnv)
+	if user == "" || pswd == "" {
+		fmt.Print("Username:")
+		fmt.Scanf("%s", &user)
```
Comment (jacobhaven) [inline cmd/ro/main.go]: Isn't this equivalent to `fmt.Scan(&user)`?

Comment (lziest) [inline cmd/ro/main.go]: nice. I will fix it.

File: cmd/ro/main.go [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+		var err error
+		pswd, err = gopass.GetPass("Password:")
+		processError(err)
+	}
+}
+
+func processError(err error) {
+	if err != nil {
+		fmt.Println("error:", err)
```
Comment (jacobhaven) [inline cmd/ro/main.go]: This should probably print to `stderr`.
`fmt.Fprintln(os.Stderr,...)`
Or just call `log.Fatal("error:",err)`

**Comment by jacobhaven** [comment]
Other than those nits, LGTM.

**Comment by grittygrease** [comment]
The inputs and outputs of the CLI are JSON. That's not ideal.

File: cmd/ro/main.go [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+var action, user, pswd, userEnv, pswdEnv, server, caPath string
+
+var owners, lefters, righters, inPath, labels, outPath, outEnv string
+
+var uses int
+
+var time, users string
+
+func registerFlags() {
+	flag.StringVar(&action, "action", "", "client action")
```
Comment (kisom) [inline cmd/ro/main.go]: Maybe having the action as the first (non-flag) argument would make this easier to use?

**Comment by kisom** [comment]
Any thoughts on a config file? There's a lot of arguments that won't change between invocations and will be repetitive to type.

**Comment by kisom** [comment]
LGTM.

**Pull request merged**